lib/buildenvsetup/ceconan-rust.ts

```diff
@@ -48,11 +48,15 @@ export class BuildEnvSetupCeConanRustDirect extends BuildEnvSetupCeConanDirect {
     }
 
     getArchFromTriple(triple) {
```

Comment:
```suggestion
    getArchFromTriple(triple: string): string {
```

Reply:
yeah but the person who is on the issue said to check and return a blank string: https://github.com/compiler-explorer/compiler-explorer/issues/4456#issuecomment-1382856240

Reply:
Thoughts @partouf? I think it's certainly better if we keep getArchFromTriple clean of any bad inputs